LibreHardwareMonitor/UI/MainForm.cs

```diff
@@ -283,6 +284,19 @@ public MainForm()
                     Server.StopHttpListener();
             };
 
+            _compressJSONData = new UserOption("compressJSONData", true, dataCompressionWebServerMenuItem, _settings);
```

Comment:
I don't think this has to be an option, the accept header is already checked, so if it's accepting gzip, it can send it out.